src/schema/resolvers.js

```diff
@@ -748,12 +760,35 @@ module.exports = {
   },
 
   Topic: {
-    oracles: async ({ address }, data, { db: { Oracles } }) => Oracles.find({ topicAddress: address }),
-    transactions: async ({ address }, data, { db: { Transactions } }) => Transactions.find({ topicAddress: address }),
+    oracles: ({ address }, data, { db: { Oracles } }) => Oracles.find({ topicAddress: address }),
+    transactions: async ({ address }, data, { db: { Transactions } }) => {
+      const types = [{ type: 'WITHDRAWESCROW' }, { type: 'WITHDRAW' }];
+      return Transactions.find({ topicAddress: address, $or: types });
+    },
   },
 
   Oracle: {
-    transactions: async ({ address }, data, { db: { Transactions } }) => Transactions.find({ oracleAddress: address }),
+    transactions: (oracle, data, { db: { Transactions } }) => {
+      const phase = getPhase(oracle);
+      let types = [];
+      switch (phase) {
+        case 'betting':
```

Comment:
can you wrap each case in brackets { }

Reply:
I can. Do you mind if we do it this way though? I just did a quick [google search](https://www.google.com/search?q=js+most+common+way+of+doing+switch+statements&oq=js+most+common+way+of+doing+switch+statements&aqs=chrome..69i57j69i64l2.7190j0j1&sourceid=chrome&ie=UTF-8) and this way is the most common way. Also, [all the examples on W3Schools](https://www.w3schools.com/jsref/jsref_switch.asp) only use `{}` on the initial `switch`

Reply:
Not a big deal, just thought the most common way might be better? :)

Reply:
Hm interesting. Guess they are following python style.